src/services/rns/rns.processor.ts

```diff
@@ -9,54 +9,45 @@ import Utils from 'web3-utils'
 import { Eth } from 'web3-eth'
 import { getBlockDate } from '../../blockchain/utils'
 import { Logger } from '../../definitions'
+import DomainExpiration from './models/expiration.model'
+import config from 'config'
 
-async function transferHandler (logger: Logger, eventData: EventData): Promise<void> {
```

Comment:
Not sure what you are referring to here. Is it the space? that's my formatter. I guess I should fix it because it adds the space everywhere I visit making PRs less clear.

Reply:
Yeah the space :-) 
Sounds good :-)